cfpb/capital-framework
# Add -10 level colors to palette and use in notification backgrounds

In the course of updating our link hover color for accessible contrast, we observed that even the base link color is inaccessible on our current -20 level notification background colors. This commit adds -10 level shades to all colors (except Gray, which already had one) and updates the notification background colors to all be one shade lighter.

/cc @designlanguage

## Additions

- Color variables for new -10 level shades in all colors
- Purple colors added to chart in `brand-palette.less`'s opening comments

## Changes

- Notification backgrounds colors changed to one step lighter than before


## Testing

Follow [the steps to testing with the docs](https://github.com/cfpb/capital-framework/blob/master/CONTRIBUTING.md#testing-in-the-documentation-site). Presuming you have Bundler set up already:

1. Pull this branch
1. `gulp`
1. `cd docs`
1. `bundle exec jekyll serve`
1. Visit http://localhost:4000/components/cf-notifications/#action-notification and see the new lighter background colors (cf. https://cfpb.github.io/capital-framework/components/cf-notifications/#action-notification)


## Screenshots

### Before

![Screenshot of cf-notification docs with current background colors](https://user-images.githubusercontent.com/1044670/60351696-472f4300-9994-11e9-92a7-93fcd18330ea.png)

### After

![Screenshot of cf-notification docs showing new lighter background colors](https://user-images.githubusercontent.com/1044670/60351663-341c7300-9994-11e9-8e6d-78751f2c77dd.png)


## Checklist

- [x] PR has been tagged with the type of change it includes
      (breaking, enhancement, bug, documentation or internal)
- [x] Project documentation has been updated in this PR with the `gulp docs` command
